src/pheval/prepare/create_spiked_vcf.py

```diff
@@ -123,11 +124,15 @@ def parse_assembly(self) -> tuple[str, bool]:
         for line in self.vcf_contents:
             if line.startswith("##contig=<ID"):
                 line_split = line.split(",")
-                chromosome = line_split[0].split("=")[2]
+                chromosome = [line for line in line_split if "ID=" in line][0].split("ID=")[1]
                 if "chr" in chromosome:
                     chr_status = True
                     chromosome = chromosome.replace("chr", "")
-                contig_length = line_split[1].split("=")[1]
+                contig_length = re.sub(
+                    "[^0-9]+",
+                    "",
+                    [line for line in line_split if "length=" in line][0].split("length=")[1],
+                )
```

Comment:
I would still need to use indices to retrieve the contig length, i.e:

```python
[line for line in line_split if "length=" in line][0]
```

I don't think it would be that easy to use regex to remove the content as other fields can be contained in the contig field (like a URL). In this case, I think it is easier to retrieve the length in this way. However, I can use regex to remove `length=` from the string once retrieved, avoiding using an index in that step